### PR TITLE
Preview issue

### PR DESCRIPTION
**Pull request by pankajhmcts**
### Change description ###
Changes to fix preview env issue

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: charts/adoption-cos-api/values.preview.template.yaml [hunk 1]
```diff
@@ -85,7 +85,7 @@ ccd:
         CCD_DOCUMENT_URL_PATTERN: ^https?://(((?:api-gateway\.preprod\.dm\.reform\.hmcts\.net|dm-store-aat\.service\.core-compute-aat\.internal(?::\d+)?)\/documents\/[A-Za-z0-9-]+(?:\/binary)?)|(em-hrs-api-aat\.service\.core-compute-aat\.internal(?::\d+)?\/hearing-recordings\/[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}\/segments\/[0-9]))
         IDAM_API_BASE_URL: https://idam-api.aat.platform.hmcts.net
         IDAM_OIDC_URL: https://idam-web-public.aat.platform.hmcts.net
-        CASE_DOCUMENT_AM_URL: http://ccd-case-document-am-api-adoption-cos-api-pr-${CHANGE_ID}.service.core-compute-preview.internal
+        CASE_DOCUMENT_AM_URL: https://ccd-case-document-am-api-adoption-cos-api-pr-${CHANGE_ID}.service.core-compute-preview.internal
```
Comment (adusumillipraveen) [inline charts/adoption-cos-api/values.preview.template.yaml:88]: this should be through svc URLs.

File: charts/adoption-cos-api/values.preview.template.yaml [hunk 2]
```diff
@@ -188,7 +188,7 @@ xui-webapp:
       HEALTH_TERMS_AND_CONDITIONS_API: http://xui-terms-and-conditions-${SERVICE_FQDN}.service.core-compute-preview.internal/health
       SERVICES_CCD_COMPONENT_API: http://gateway-${SERVICE_FQDN}
       SERVICES_CCD_DATA_STORE_API: http://${SERVICE_NAME}-ccd-data-store-api
-      SERVICES_DOCUMENTS_API_V2: http://ccd-case-document-am-api-adoption-cos-api-pr-${CHANGE_ID}.service.core-compute-preview.internal
+      SERVICES_DOCUMENTS_API_V2: https://ccd-case-document-am-api-adoption-cos-api-pr-${CHANGE_ID}.service.core-compute-preview.internal
```
Comment (adusumillipraveen) [inline charts/adoption-cos-api/values.preview.template.yaml:191]: this should be through svc URLs.

File: charts/adoption-cos-api/values.preview.template.yaml [hunk 3]
```diff
@@ -205,7 +205,7 @@ ccd-case-document-am-api:
     environment:
       CASE_DOCUMENT_S2S_AUTHORISED_SERVICES: ccd_case_document_am_api,ccd_gw,xui_webapp,ccd_data,bulk_scan_processor,dg_docassembly_api,bulk_scan_orchestrator,adoption_cos_api,adoption_web
       DM_STORE_BASE_URL: http://dm-store-aat.service.core-compute-aat.internal
-      CCD_DATA_STORE_API_BASE_URL: http://ccd-data-store-api-adoption-cos-api-pr-${CHANGE_ID}.service.core-compute-preview.internal
+      CCD_DATA_STORE_API_BASE_URL: https://ccd-data-store-api-adoption-cos-api-pr-${CHANGE_ID}.service.core-compute-preview.internal
```
Comment (adusumillipraveen) [inline charts/adoption-cos-api/values.preview.template.yaml:208]: this should be through svc URLs.

Comment (hannah38) [inline charts/adoption-cos-api/values.preview.template.yaml:208]:
```suggestion
      CCD_DATA_STORE_API_BASE_URL: http://adoption-cos-api-pr-${CHANGE_ID}-ccd-data-store-api
```

File: charts/adoption-cos-api/values.preview.template.yaml (hunk 2 above)
Comment (hannah38) [inline charts/adoption-cos-api/values.preview.template.yaml:191]:
```suggestion
      SERVICES_DOCUMENTS_API_V2: http://adoption-cos-api-pr-${CHANGE_ID}-ccd-case-document-am-api
```

File: charts/adoption-cos-api/values.preview.template.yaml (hunk 1 above)
Comment (hannah38) [inline charts/adoption-cos-api/values.preview.template.yaml:88]:
```suggestion
        CASE_DOCUMENT_AM_URL: http://adoption-cos-api-pr-${CHANGE_ID}-ccd-case-document-am-api
```

File: Jenkinsfile_CNP [hunk 4]
```diff
@@ -190,12 +190,13 @@ withPipeline(type, product, component) {
   }
 
   onPR {
+    env.NODE_TLS_REJECT_UNAUTHORIZED = "0";
     env.ENVIRONMENT="preview"
-    env.CASE_API_URL = "http://adoption-cos-api-pr-${CHANGE_ID}.service.core-compute-preview.internal"
+    env.CASE_API_URL = "https://adoption-cos-api-pr-${CHANGE_ID}.service.core-compute-preview.internal"
```
Comment (adusumillipraveen) [inline Jenkinsfile_CNP:195]:
```suggestion
    env.CASE_API_URL = "http://adoption-cos-api-pr-${CHANGE_ID}-java"
```

**Pull request merged**